Altis_Life.Altis/stringtable.xml

```diff
@@ -4071,7 +4071,7 @@
         <Key ID="STR_NOTF_CannotSeizePerson">
             <Original>You couldn't effectively seize any items. Try again.</Original>
             <Czech>Dalo by se efektivně využily všech položek. Zkus to znovu.</Czech>
-            <French>Vous ne pouviez pas saisir les articles. Réessayez.</French>
+            <French>Vous ne pouvez pas saisir les objets. Réessayez.</French>
```

Comment:
"You couldn't effectively seize any items. Try again."
You changed the sense of the sentence here. Need to be turned in another way.